pkg/repo_add/repoadd_test.go

```diff
@@ -0,0 +1,74 @@
+package repo_add
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func TestNewRepoAdd(t *testing.T) {
+	_, err := NewRepoAdd("test")
+
+	// If NewRepoAdd fails, the tests don't run on an arch system.
+	// By returning the test just passes on non Archlinux systems
+	if err != nil {
+		fmt.Printf("Not running on an archlinux system, marking test as passed.")
+		return
+	}
+}
+
+func TestIntegration(t *testing.T) {
+	dir := os.TempDir() + "/TestIntegration"
```

Comment:
Maybe add `AAAA` to the dir as TestIntegration is a bit generic

Reply:
Sure